examples/time_frequency/plot_time_frequency_erds.py

```diff
@@ -111,3 +109,81 @@
     fig.colorbar(axes[0].images[-1], cax=axes[-1])
     fig.suptitle("ERDS ({})".format(event))
     fig.show()
+
+##############################################################################
+# Similar to `~mne.Epochs` objects, we can also export data from
+# `~mne.time_frequency.EpochsTFR` and `~mne.time_frequency.AverageTFR` objects
+# to a :class:`Pandas DataFrame <pandas.DataFrame>`:
+
+df = tfr.to_data_frame(time_format=None)
+df.head()
+
+##############################################################################
+# This allows us to use additional plotting functions like
+# :func:`seaborn.lineplot` to easily plot confidence bands:
+
+df = tfr.to_data_frame(time_format=None, long_format=True)
+df['channel'].cat.reorder_categories(['C3', 'Cz', 'C4'], ordered=True,
+                                     inplace=True)
+
+freq_bands = {'delta': (0.5, 4),
+              'theta': (5, 7),
+              'alpha': (8, 14),
+              'beta': (15, 35)}
+
+
+def map_bands(freq):
+    for band, (low_lim, high_lim) in freq_bands.items():
+        if low_lim <= freq <= high_lim:
+            return band
+
+
+df['band'] = pd.Categorical(
+    df['freq'].map(map_bands),
+    categories=['beta', 'alpha', 'theta', 'delta'],
+    ordered=True)
+
+g = sns.FacetGrid(df, row='band', col='channel', margin_titles=True)
+g.map(sns.lineplot, 'time', 'value', 'condition', n_boot=10)
+for ax in g.axes.ravel():
+    ax.axhline(0, color='black', linestyle='dashed', linewidth=0.5,
+               alpha=0.5)
+    ax.axvline(0, color='black', linestyle='dashed', linewidth=0.5,
+               alpha=0.5)
+    ax.set(ylim=(None, 1.5))
```

Comment:
these can also use `g.map()` to eliminate the for-loop, for example:

``py
g.map(plt.axhline, y=0, color='black', linestyle='dashed', linewidth=0.5,
      alpha=0.5)
```

Reply:
for ylim, you have `g.set(ylim=(None, 1.5))`

Reply:
thanks! Indeed way nicer. Both implemented.